gnosis/safe-react
# Feature/2517 use string for chain

## What it solves
Resolves #2517 

## How this PR fixes it
Changes number for ETHEREUM_NETWORK (enum with strings as values)

## How to test

* Check that the address book import/export still works across several chains
* Check that the Safe Apps are still filtered per chain
